pypeman/plugins/tests/test_remoteadmin.py

```diff
@@ -102,9 +104,13 @@ async def test_list_channels(self, webremoteclient):
 
         assert resp.status == 200
         json_resp = json.loads(await resp.text())
-        assert len(json_resp) == 1
+        print(json_resp)
```

Comment:
print?

Reply:
apologies. didn't notice this is a test. up to you to decide if it helps for debugging tests

Reply:
removed